### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by SPalominos**
Improve code quality

### CODE REVIEW

File: geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy [hunk 1]
```diff
@@ -21,6 +19,11 @@ import org.orbisgis.processmanagerapi.IProcess
  * @author Jérémy Bernard
  */
 static IProcess holeAreaDensity() {
+    //Definition of constant values
+    def final GEOMETRIC_FIELD = "the_geom"
```
Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy]: Does that make sense to put it out the closure ?

File: geoindicators/src/main/groovy/org/orbisgis/DataUtils.groovy [hunk 2]
```diff
@@ -14,52 +14,49 @@ import org.orbisgis.processmanagerapi.IProcess
...
+    processFactory.create("Utility process to join tables in one",
+            [inputTableNamesWithId: Map, outputTableName: String, datasource: JdbcDataSource],
+            [outputTableName: String],
             { inputTableNamesWithId, outputTableName, JdbcDataSource datasource ->
 
                 logger.info("Executing Utility process to join tables in one")
 
-                String columnKey
-                String a = "a"
+                def columnKey
```
Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/DataUtils.groovy]: If yes. All constants should be extracted.
Not sure it adds anything to the readability of the code

File: geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy (hunk 1 above)
Comment (SPalominos) [reply geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy]: Making them as constants ensure that those variable won't be changed and it's also a good practice to extract constants.

Comment (ebocher) [reply geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy]: Not sure to be aligned with this argument ;-)

File: geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy [hunk 3]
```diff
@@ -92,116 +94,122 @@ return processFactory.create(
...
+    def final ID_COLUMN_RSU = "id_rsu"
+    def final HEIGHT_WALL = "height_wall"
+    def final BASE_NAME = "rsu_ground_sky_view_factor"
+    
     return processFactory.create(
             "RSU ground sky view factor",
-            [rsuTable: String,correlationBuildingTable: String, rsuBuildingDensityColumn: String, pointDensity: double, rayLength: double,
-             numberOfDirection: int, prefixName: String, datasource: JdbcDataSource],
+            [rsuTable: String,correlationBuildingTable: String, rsuBuildingDensityColumn: String, pointDensity: 0.008D,
+                    rayLength: 100D, numberOfDirection: 60, prefixName: String, datasource: JdbcDataSource],
```
Comment (j3r3m1) [inline geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: why D after the default values ? For Double ?

File: geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy [hunk 4]
```diff
@@ -460,108 +478,110 @@ static IProcess projectedFacadeAreaDistribution() {
...
-
                 // To avoid overwriting the output files of this step, a unique identifier is created
-                def uid_out = UUID.randomUUID().toString().replaceAll("-", "_")
-
                 // Temporary table names
-                def buildRoofSurfIni = "build_roof_surf_ini" + uid_out
-                def buildVertRoofInter = "build_vert_roof_inter" + uid_out
-                def buildVertRoofAll = "buildVertRoofAll" + uid_out
-                def buildRoofSurfTot = "build_roof_surf_tot" + uid_out
+                def buildRoofSurfIni = "build_roof_surf_ini" + uuid()
```
Comment (j3r3m1) [inline geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: Is uuid() a function that you have coded for the need of geoclimate ? Or an existing one ?

File: geoindicators/src/main/groovy/org/orbisgis/Geoclimate.groovy [hunk 5]
```diff
@@ -5,26 +5,23 @@ import org.orbisgis.processmanagerapi.IProcessFactory
...
+ * Root access point to the Geoclimate processes.
+ */
 abstract class Geoclimate extends Script {
 
     public static IProcessFactory processFactory = ProcessManager.getProcessManager().factory("geoclimate")
-
     public static Logger logger = LoggerFactory.getLogger(Geoclimate.class)
 
-    public static BuildingIndicators = new BuildingIndicators()
+    static def uuid(){UUID.randomUUID().toString().replaceAll("-", "_")}
```
Comment (j3r3m1) [inline geoindicators/src/main/groovy/org/orbisgis/Geoclimate.groovy]: ok I understand now...

**Pull request merged**